### PR TITLE
<Checkbox/> - Implement selection area

### PR DESCRIPTION
**Pull request by EgoziE**
feature - https://github.com/wix/wix-style-react/issues/4126

### CODE REVIEW

File: src/Checkbox/test/Checkbox.visual.js [hunk 1]
```diff
@@ -53,6 +53,10 @@ const tests = [
         it: 'error',
         props: { hasError: true },
       },
+      {
```
Comment (mosheKerbel) [inline src/Checkbox/test/Checkbox.visual.js]: try adding hover tests for all combinations:
1. hover on "always"
2. hover on "always" that is checked
2. hover on "hover"
3. hover on "hover" that is checked

**Comment by mosheKerbel** [comment]
test3:e2e is failing over **checkbox** tests, check it out

File: src/Checkbox/test/Checkbox.visual.js (hunk 1 above)
Comment (EgoziE) [reply src/Checkbox/test/Checkbox.visual.js]: we have a problem with our driver, will open an issue

**Comment by mosheKerbel** [comment]
Merging it, since we fail here for the same unrelated test that fail on master.

**Comment by mosheKerbel** [comment]
@EgoziE add to changelog

**Pull request merged**